packages/blade/src/components/Icons/_Svg/Svg/Svg.native.tsx

```diff
@@ -1,19 +1,29 @@
 import type { ReactElement } from 'react';
 import { Svg as SvgNative } from 'react-native-svg';
 import type { SvgProps } from './types';
-import { makeAccessible } from '~utils';
+import { getStyledProps, makeAccessible } from '~utils';
+import BaseBox from '~components/Box/BaseBox';
 
-const Svg = ({ children, height, viewBox, width, fill }: SvgProps): ReactElement => {
+const Svg = ({
+  children,
+  height,
+  viewBox,
+  width,
+  fill,
+  ...styledProps
+}: SvgProps): ReactElement => {
   return (
-    <SvgNative
-      {...makeAccessible({ hidden: true })}
-      height={height}
-      viewBox={viewBox}
-      width={width}
-      fill={fill}
-    >
-      {children}
-    </SvgNative>
+    <BaseBox {...getStyledProps(styledProps)}>
+      <SvgNative
+        {...makeAccessible({ hidden: true })}
+        height={height}
+        viewBox={viewBox}
+        width={width}
+        fill={fill}
+      >
+        {children}
+      </SvgNative>
+    </BaseBox>
```

Comment:
This code reads props from `<Svg` component and adds it to `BaseBox` but changes in all the icons to pass these props to `Svg` component are in separate PR (they were too many changes and were making the review difficult)

Reply:
Why are we wrapping it in a box? can't we add the styling directly to the root svg element?

Reply:
For Svg, I added styling to root element now. Still keeping the Box wrappers on Button and some other block elements since it's easier to add and maintain.